ee/codegen/src/__test__/nodes/__snapshots__/tool-calling-node.test.ts.snap

```diff
@@ -0,0 +1,125 @@
+        )
+    }
+    display_data = NodeDisplayData(
+        position=NodeDisplayPosition(x=0, y=0), width=None, height=None
+    )
+"
+`;
+
+exports[`ToolCallingNode > basic > getNodeFile 1`] = `
+"from vellum.workflows.nodes.experimental.tool_calling_node import (
+    ToolCallingNode as BaseToolCallingNode,
+)
+
+
+class ToolCallingNode(BaseToolCallingNode):
+    ml_model = "gpt-4o-mini"
+    blocks = [
+        {
+            "state": None,
+            "blocks": [
+                {
+                    "state": None,
+                    "blocks": [
+                        {
+                            "text": "You are a weather expert",
+                            "state": None,
+                            "block_type": "PLAIN_TEXT",
+                            "cache_config": None,
+                        },
+                    ],
+                    "block_type": "RICH_TEXT",
+                    "cache_config": None,
+                },
+            ],
+            "chat_role": "SYSTEM",
+            "block_type": "CHAT_MESSAGE",
+            "chat_source": None,
+            "cache_config": None,
+            "chat_message_unterminated": None,
+        },
+        {
+            "state": None,
+            "blocks": [
+                {
+                    "state": None,
+                    "blocks": [
+                        {
+                            "state": None,
+                            "block_type": "VARIABLE",
+                            "cache_config": None,
+                            "input_variable": "question",
+                        },
+                    ],
+                    "block_type": "RICH_TEXT",
+                    "cache_config": None,
+                },
+            ],
+            "chat_role": "USER",
+            "block_type": "CHAT_MESSAGE",
+            "chat_source": None,
+            "cache_config": None,
+            "chat_message_unterminated": None,
+        },
+    ]
+    functions = [
+        {
+            "name": "get_current_weather",
+            "state": None,
+            "forced": None,
+            "strict": None,
+            "parameters": {
+                "type": "object",
+                "required": [
+                    "location",
+                    "unit",
+                ],
+                "properties": {
+                    "unit": {
+                        "type": "string",
+                    },
+                    "location": {
+                        "type": "string",
+                    },
+                },
+            },
+            "description": None,
+            "cache_config": None,
+        },
+    ]
+    prompt_inputs = {
+        "question": "What's the weather like in San Francisco?",
+    }
+    max_tool_calls = 1
+
+    class NodeTrigger(BaseToolCallingNode.Trigger):
+        merge_behavior = "AWAIT_ALL"
+
+    class Outputs(BaseToolCallingNode.Outputs):
+        text = ""
+        chat_history = []
```

Comment:
[blocking] we shouldn't need to define these, and most importantly, I don't think they start out with default values for them both like this